rotkehlchen/tests/utils/ethereum.py

```diff
@@ -342,12 +343,16 @@ def get_decoded_events_of_transaction(
         database: DBHandler,
         tx_hash: EVMTxHash,
         transactions: EvmTransactions | None = None,
+        relevant_address: ChecksumAddress | None = None,
 ) -> tuple[list['EvmEvent'], EVMTransactionDecoder]:
     """A convenience function to ask get transaction, receipt and decoded event for a tx_hash
 
     It also accepts `transactions` in case the caller whants to apply some mocks (like call_count)
     on that object.
 
+    If relevant_address is provided then the transactions add added linked to
```

Comment:
wording makes no sense

```suggestion
    If relevant_address is provided then the added transactions are added linked to
```